catroid/src/main/java/org/catrobat/catroid/io/StorageHandler.java

```diff
@@ -1417,14 +1417,11 @@ private void copyFile(InputStream in, OutputStream out) throws IOException {
 	}
 
 	public void unzipTemplate(String projectName, String templateName, String zipFileName, Activity activity) {
```

Comment:
Well, there are more zip-methods in the StorageHandler. I've moved them to UtilZip.